src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -202,10 +218,14 @@ def run(self, revision):
 
         if not issues:
             logger.info('No issues, stopping there.')
+            self.index(revision, state='done', issues=0)
             return
 
         # Report issues publication stats
-        stats.api.increment('analysis.issues.publishable', len([i for i in issues if i.is_publishable()]))
+        nb_issues = len(issues)
+        nb_publishable = len([i for i in issues if i.is_publishable()])
+        self.index(revision, state='analyzed', issues=nb_issues, issues_publishable=nb_publishable)
```

Comment:
Maybe we are using the index service not exactly in its intended use case as we're storing data of the tasks too and we're using it to report progress.
In the future, we might want to look into the [taskcluster-treeherder](https://docs.taskcluster.net/docs/reference/integrations/taskcluster-treeherder) service.